guides/common/modules/con_smartproxy-networking.adoc

```diff
@@ -0,0 +1,40 @@
+[id="{SmartProxy}-Networking_{context}"]
+= {SmartProxy} networking
+
+The communication between {ProjectServer} and managed hosts registered to a {SmartProxyServer} is routed through that {SmartProxyServer}.
+{SmartProxyServer} can also provide {Project} services to managed hosts.
+
+Many of the services that {SmartProxyServer} manages use dedicated network ports.
+However, {SmartProxyServer} ensures that all communications from the host to {ProjectServer} use a single source IP address, which simplifies firewall administration.
+
+.{Project} topology with hosts connecting to a {SmartProxy}
+
+In this topology, {SmartProxy} provides a single endpoint for all host network communications so that in remote network segments, only firewall ports to the {SmartProxy} itself must be open.
+
+ifndef::satellite[]
+include::snip_red-hat-images.adoc[]
+endif::[]
+
+.How {Project} components interact when hosts connect to a {SmartProxy}
+image::topology-isolated-satellite.png[{ProjectName} topology with an isolated host]
+
+.{Project} topology with hosts connecting directly to {ProjectServer}
+
+In this topology, hosts connect to {ProjectServer} rather than an isolated {SmartProxy}.
+This situation applies also to {SmartProxies} themselves because the external {SmartProxy} is a client of {ProjectServer}.
```

Comment:
Please reword "this situation".

also "client" > host

external SP > SmartProxyServer

Reply:
Addressed.